JM/CF/Scripts/1_Core/CommunityFramework/Logging/CF_Log.c

```diff
@@ -1,6 +1,6 @@
 class CF_Log
 {
-	static int Level = CF_LogLevel.ERROR;
+	static int Level = CF_LogLevel.TRACE;
```

Comment:
The default level should remain Error. I guess this was a testing leftover?

Reply:
Correct